technologies/job/bash/debian9-stretch/image_test.yaml

```diff
@@ -18,6 +18,10 @@ fileExistenceTests:
     path: "/entrypoint"
     shouldExist: true
     permissions: "-rwxr-xr-x"
+  - name: "kinit"
```

Comment:
As we consider all previous tests are copied in other images (gcp, azure and aws), we should also move kerberos tests in all image_test.yaml files for bash techno, isn't it ?